src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Attributes.cs

```diff
@@ -129,4 +129,53 @@ public SetWin32ContextInIDispatchAttribute()
         {
         }
     }
+
+    [AttributeUsage(AttributeTargets.Interface, Inherited = false)]
+    public sealed class TypeLibImportClassAttribute : Attribute
+    {
+        internal String _importClassName;
+        public TypeLibImportClassAttribute(Type importClass)
+        {
+            _importClassName = importClass.ToString();
+        }
+        public String Value { get { return _importClassName; } }
```

Comment:
Nit: you could simplify this and other such cases like:
```C#
public TypeLibImportClassAttribute(Type importClass)
{
    Value = importClass;
}
public string Value { get; }
```
(Also note the changing of String to string, to conform to corefx guidelines)